likeminds-chat-reactnative-integration/ChatSX/components/InputBox/models/InputBoxProps.ts

```diff
@@ -19,4 +19,5 @@ export interface InputBoxProps {
   chatroomName?: any;
   currentChatroomTopic?: Conversation;
   isGif?: boolean;
+  widgets: Record<string, any> | {}
```

Comment:
define types.

Reply:
Made this to `Record<string, any>` as this is meta data object that we will pass